provider.go

```diff
@@ -14,9 +15,13 @@ type Provider struct {
 	APIToken string `json:"auth_token"`
 }
 
+func (p *Provider) getDomain(fqdn string) string {
```

Comment:
Hmm, this doesn't actually _get_ anything... as weird as this seems, how about this instead:

```suggestion
// unFQDN trims any trailing "." from fqdn. DigitalOcean's API does not use FQDNs.
func (p *Provider) unFQDN(fqdn string) string {
```

Reply:
@mholt Good suggestion, applied! I have verified that it didn't work before and now does using the `_example/main.go` file, so I assume it will also work with Caddy

Reply:
Good, now just please make sure it compiles. Thanks!

Reply:
My bad, fixed!